translate/rollup.config.mjs

```diff
@@ -10,7 +10,7 @@ import css from 'rollup-plugin-css-only';
 /** @type {import('rollup').RollupOptions} */
 const config = {
   input: 'src/index.tsx',
-  output: { file: 'dist/translate.js' },
+  output: { file: 'dist/translate.js', format: 'iife' },
```

Comment:
One of the new dependencies includes a module with a top-level statement `const top = ...`. With the default `output.format`, Rollup presumes that the code will be included in a `<script type=module>` so leaves it at the top level. However, as we don't do that, the output needs to be wrapped in an IIFE.